packages/salesforcedx-vscode-core/test/vscode-integration/util/cliConfiguration.test.ts

```diff
@@ -94,43 +95,35 @@ describe('SFDX CLI Configuration utility', () => {
 
     it('Should return true if cli is not installed', async () => {
       getConfigValueStub.withArgs(SFDX_CONFIG_DISABLE_TELEMETRY).returns('');
+      sandboxStub
+        .stub(ConfigAggregator.prototype, 'getPropertyValue')
```

Comment:
dumb q: can we stub this in the before each?

Reply:
I updated as to use the before each and removed one of the tests since isCLITelemetryAllowed only returns true or false.